### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SergiroZ**
LabWork_1

### CODE REVIEW

**Comment by SergiroZ** [comment]
LabWork_1_(mod.)_

File: Zuenok/BookLibraryCRUD/BookLibraryCRUD/Source/BookService.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        public BookService(ILibrary repository)
+        {
+            if (repository != null) this.repository = repository;
+            Books = repository?.GetBooks();
+        }
+
+        /// <summary>
+        ///     All books from book repository
+        /// </summary>
+        public IEnumerable<Book> Books { get; }
```
Comment (Levchenkov) [inline Zuenok/BookLibraryCRUD/BookLibraryCRUD/Source/BookService.cs]: This is not the best idea to make public access to the all object via public property. It's better to make method GetAll()

File: Zuenok/BookLibraryCRUD/BookLibraryCRUD/Source/BookService.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            return book ?? repository.GetLast();
+        }
+
+        /// <summary>
+        ///     Getting ID of the last item in the list of books
+        /// </summary>
+        /// <returns>ID code</returns>
+        public int GetLastId()
+        {
+            var res = repository.GetLast()?.Id;
```
Comment (Levchenkov) [inline Zuenok/BookLibraryCRUD/BookLibraryCRUD/Source/BookService.cs]: У последнего элемента не всегда максимальный Id

File: Zuenok/BookLibraryCRUD/UnitTestBookLibrary/BookLibraryTest.cs [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+using BookLibraryCRUD;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
+
+namespace UnitTestBookLibrary
+{
+    [TestClass]
+    public class LibraryDBInitializatorTest
```
Comment (Levchenkov) [inline Zuenok/BookLibraryCRUD/UnitTestBookLibrary/BookLibraryTest.cs]: Fiel name should be the same as class name

**Pull request merged**